### DISCUSSION TITLE
How to document private methods

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/docfx#10045] [Q&A]
DocFx ver. 2.76.0

DocFx's Filter APIs states the following
>To show private methods, set the config to . When enabled, internal only langauge keywords such as or starts to appear in the >declaration of all APIs, to accurately reflect API accessibility.includePrivateMemberstrueprivateinternal

After much trial and error, I was unable to document the private method.
I would like to create a complete document including private methods for my team members, can anyone help me with the procedure?
If possible, please let me know the flow, including file names that must be edited.

**Comment by TSS-22** [comment]
You need to add the tag `"includePrivateMembers": true,` to `metadata` such as:

```
"metadata": [
    {
      "src": [
        {
          "src": "..",
          "files": ["My/Project/*.csproj"],
          "exclude": ["**/bin/**", "**/obj/**"]
        }
      ],
      "dest": "api",
      "includePrivateMembers": true,
    }
  ],
```
